.gitignore

```diff
@@ -0,0 +1,6 @@
+/.idea/homebrew-zulu.iml
```

Comment:
You can ignore base `.idea` directory without adding each file inside it. Just add `.idea/`.